app/views/unattended/provisioning_templates/snippet/pxegrub2_chainload.erb

```diff
@@ -43,7 +42,10 @@ echo "the grub2_chainload template."
 echo
 echo "(*) grub2-efi-x64-2.02-122.el8 (upstream doesn't have the patches yet)"
 echo
-#connectefi scsi
+<%=
+connectefi_option = host_param('pxegrub_connectefi_option')
+"connectefi #{connectefi_option}" if connectefi_option
+-%>
```

Comment:
According to https://github.com/rhboot/grub2/blob/72b17102c157707acfffb01ae7694de6cf23eb5c/grub-core/commands/efi/connectefi.c#L114 it can be `scsi` or `pciroot`.